.github/workflows/coding-standards.yml

```diff
@@ -25,4 +25,4 @@ jobs:
     name: "Coding Standards"
     uses: "doctrine/.github/.github/workflows/coding-standards.yml@6.0.0"
     with:
-      php-version: "8.3"
+      php-version: "8.4"
```

Comment:
Why are we overriding the default value, here and below?

Reply:
Because in the Laminas-related projects we have always been running all tools (i.e. everything except the unit tests) with the latest PHP version supported by the library. Hence, we think the version should be coupled to the current state of the library and not to the upstream workflow.

Reply:
In practice, do those ever get out of sync for long?

Reply:
Probably not for too long, but it never got out of sync, as we note it here explicitly 😊